ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -530,6 +534,41 @@ def test_ipa_dns_systemrecords_check(self):
             assert check["result"] == "SUCCESS"
             assert check["kw"]["key"] in SRV_RECORDS
 
+    def test_ipa_healthcheck_ds_ruv_check(self):
+        """
+        This test checks if RUVCheck displays the correct result.
+        """
+        suffix = ipautil.realm_to_suffix(self.master.domain.realm)
+        dn_suffix = escape_dn_chars(str(suffix))
+        conn = self.replicas[0].ldap_connect()
+        entries = conn.get_entries(
+            DN("cn=replica", "cn=" + dn_suffix, "cn=mapping tree, cn=config"),
+            filter="(objectclass=nsds5replicationagreement)",
+            attrs_list=["nsds50ruv"],
```

Comment:
Your search base should be dn_suffix and the filter should be "(&(nsUniqueId=ffffffff-ffffffff-ffffffff-ffffffff)(objectClass=nstombstone))"

ldapsearch -LLL -x -D 'cn=directory manager' -W -b dc=example,dc=test "(&(nsUniqueId=ffffffff-ffffffff-ffffffff-ffffffff)(objectClass=nstombstone))" nsds50ruv

```
dn: cn=replica,cn=dc\3Dexample\2Cdc\3Dtest,cn=mapping tree,cn=config
nsds50ruv: {replicageneration} 5e948009000000040000
nsds50ruv: {replica 4 ldap://ipa.example.test:389} 5e948009000100040000 5e9487
 08000800040000
nsds50ruv: {replica 5 ldap://replica.example.test:389} 5e9485c4000000050000 5e
 948721000100050000
nsds50ruv: {replica 3 ldap://replica.example.test:389}
```
3 is a dangling RUV.

o=ipaca also has RUVs but depending on what you're validating testing the main one may be fine.